lib/utils/archyTree.ts

```diff
@@ -4,40 +4,64 @@ import {BlueshellState} from '../nodes/BlueshellState';
 import * as archy from 'archy';
 import {Data} from 'archy';
 
-function buildArchyTree<S extends BlueshellState, E>(node: Base<S, E>, state?: S): Data {
+function buildArchyTree<S extends BlueshellState, E>(
+	node: Base<S, E>, state?: S, contextDepth = Number.MAX_SAFE_INTEGER
```

Comment:
does this function need a default contextDepth? Its not exported and is only accessed by serializeArchyTree()